test/views/payments/NoPaymentsViewSpec.scala

```diff
@@ -145,18 +145,6 @@ class NoPaymentsViewSpec extends ViewBaseSpec {
 
   "Rendering the no payments page" should {
 
-    "display the covid message" when {
```

Comment:
We still should be testing to see if the covid message is present here as it'll still be on this page